yarn.lock

```diff
@@ -2828,6 +2828,20 @@
     lz-string "^1.4.4"
     pretty-format "^26.6.2"
 
+"@testing-library/dom@^9.3.1":
```

Comment:
we  don't need these changes now since the package.json changes went away.

Reply:
Done - ty!